db/db_impl_write.cc

```diff
@@ -471,7 +467,7 @@ Status DBImpl::PipelinedWriteImpl(const WriteOptions& write_options,
         write_options.ignore_missing_column_families, 0 /*log_number*/, this,
         true /*concurrent_memtable_writes*/);
     if (write_thread_.CompleteParallelMemTableWriter(&w)) {
-      MemTableInsertStatusCheck(w.status);
+      SetBackgroundErrorIfNeeded(w.write_group->status);
```

Comment:
What happens to w.status? Previously it could set the background error but now it is simply returned to the user.

Reply:
Do we need to set the write_group status in PipelinedWriteImpl the way we do for WriteImpl?

Reply:
yeah, I updated CompleteParallelMemTableWriter to set write_group status on CompleteParallelMemTableWriter, same as in PiplinedWriteImpl.